ui/src/app/job-list/table/table.component.html

```diff
@@ -95,7 +79,7 @@
 
 <div class="group-options" *ngIf="selectedJobs.length > 0">
   {{ selectedJobs.length }} jobs selected
-  <button md-raised-button color="accent" class="group-button group-abort" *ngIf="currentStatusGroup==statusGroup.Active"
+  <button md-raised-button color="accent" class="group-button group-abort"
```

Comment:
Went with the easiest fix for the latter behavior. Again, this will be a UX preference so we can circle back later if this isn't the preferred behavior.